src/components/details-attribute/details-attribute.tsx

```diff
@@ -40,7 +40,7 @@ const styles: StyleRulesCallback<CssRules> = (theme: ArvadosTheme) => ({
 interface DetailsAttributeDataProps {
     label: string;
     classLabel?: string;
-    value?: string | number;
+    value?: any;
```

Comment:
I think that closer to the purpose of value prop of type 'any' is using children prop of type 'ReactNode'.
Then
```
<DetailsAttribute classLabel={classes.label} classValue={classes.value} label='Status'>
    <Chip label="Complete" className={classes.chip} />}
</DetailsAttribute >
```